src/azurePolicy/scanHelper.ts

```diff
@@ -37,7 +38,7 @@ export async function triggerOnDemandScan(): Promise<ScanCompletionPoll[]> {
 }
 
 async function triggerScan(scope: string, token: string): Promise<string> {
-  let triggerScanUrl = `https://management.azure.com${scope}/providers/Microsoft.PolicyInsights/policyStates/latest/triggerEvaluation?api-version=2019-10-01`;
+  let triggerScanUrl = `${await ManagementUrlHelper.getBaseUrl()}${scope}/providers/Microsoft.PolicyInsights/policyStates/latest/triggerEvaluation?api-version=2019-10-01`;
```

Comment:
Not sure whether all environments support the API versions being used for the different APIs. Good to test E2E!